contracts/test/bond/DaoConfigurationBox.sol

```diff
@@ -1,26 +1,26 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-import "../../bond/DaoBondConfiguration.sol";
+import "../../dao-configuration/DaoConfiguration.sol";
 
 /**
```

Comment:
issue: move the location of this test to match the new location of the `.sol` file it tests.